utility/src/main/java/org/oppia/android/util/parser/CustomBulletSpan.kt

```diff
@@ -21,10 +21,15 @@ class CustomBulletSpan(context: Context) : LeadingMarginSpan {
   private val bulletRadius: Int
   private val gapWidth: Int
   private val yOffset: Int
+  /** The space between the parent and the bullet. */
```

Comment:
Perhaps clearer: the space between the start of the line and the bullet (this may not actually be the parent start--there could be other styling rules including padding that affect where the span starts).

Reply:
Done